Building from git master is broken after https://github.com/bitwarden/browser/commit/2c6b3927d49fde36c8650a7cd8364025b40325e3

Seems like https://github.com/bitwarden/browser/commit/2c6b3927d49fde36c8650a7cd8364025b40325e3 broke building from git master:

```
$ npm run dist:firefox
> bitwarden-browser@0.0.0 dist:firefox /home/glitsj16/bitwarden.browser
> npm run build:prod && gulp dist:firefox


> bitwarden-browser@0.0.0 build:prod /home/glitsj16/bitwarden.browser
> gulp build && cross-env NODE_ENV=production webpack

[01:51:08] Using gulpfile ~/bitwarden.browser/gulpfile.js
[01:51:08] Starting 'build'...
[01:51:08] Finished 'build' after 878 ms
clean-webpack-plugin: /home/glitsj16/bitwarden.browser/build/* has been removed.
Hash: 97bbd99e213ea96de32b
Version: webpack 4.29.0
Time: 47206ms
Built at: 13/07/2019 01.52.01
 14 assets
Entrypoint popup/main = popup/main.js popup/main.js.map
Entrypoint background = background.js background.js.map
Entrypoint content/autofill = content/autofill.js
Entrypoint content/autofiller = content/autofiller.js
Entrypoint content/notificationBar = content/notificationBar.js
Entrypoint content/shortcuts = content/shortcuts.js
Entrypoint notification/bar = notification/bar.js notification/bar.css
Entrypoint downloader/downloader = downloader/downloader.js
Entrypoint 2fa/2fa = 2fa/2fa.js
[0] ./src/popup/main.ts 0 bytes {8} [built]
[1] ./src/background.ts 0 bytes {1} [built]
[2] ./src/content/autofill.js 41.7 KiB {2} [built]
[3] ./src/content/autofiller.ts 0 bytes {3} [built]
[4] ./src/content/notificationBar.ts 0 bytes {4} [built]
[5] ./src/content/shortcuts.ts 0 bytes {5} [built]
[6] ./src/notification/bar.js 5.13 KiB {7} [built]
[7] ./src/notification/bar.scss 41 bytes {7} [built]
[8] ./src/downloader/downloader.ts 0 bytes {6} [built]
[9] ./src/2fa/2fa.ts 0 bytes {0} [built]

ERROR in src/background/main.background.ts(208,43): error TS2554: Expected 7 arguments, but got 6.

```

--- a/src/background/main.background.ts
+++ b/src/background/main.background.ts
@@ -206,7 +206,7 @@ export default class MainBackground {
 
         if (!this.isSafari) {
             this.contextMenusBackground = new ContextMenusBackground(this, this.cipherService,
-                this.passwordGenerationService, this.analytics, this.platformUtilsService, this.lockService);
+                this.passwordGenerationService, this.analytics, this.platformUtilsService, this.lockService, this.eventService);
             this.idleBackground = new IdleBackground(this.lockService, this.storageService, this.notificationsService);
             this.webRequestBackground = new WebRequestBackground(this.platformUtilsService, this.cipherService,
                 this.lockService);

